samples/core/getting-started/unit-testing-using-dotnet-test/README.md

```diff
@@ -1,42 +1,19 @@
-Unit Testing using dotnet test Sample
-================
+# Unit testing using dotnet test sample
 
-This sample is part of the [unit testing tutorial](https://docs.microsoft.com/dotnet/articles/core/testing/unit-testing-with-dotnet-test)
-for creating applications with unit tests included. See that topic for detailed steps on the code
-for this sample.
+This sample is part of the [unit testing tutorial](https://docs.microsoft.com/dotnet/articles/core/testing/unit-testing-with-dotnet-test) for creating applications with unit tests included. See that topic for detailed steps on the code for this sample.
 
-Key Features
-------------
+## Key features
 
-This sample demonstrates creating a library and writing effective unit tests that validate the features in that library. 
+This sample demonstrates creating a library and writing effective unit tests that validate the features in that library. The example provides a service that indicates whether a number is prime.
 
-The example provides a service that indicates whether a number is prime.
+## Restore and test
```

Comment:
It's not runnable ... class lib. AFAIK one has to run the test command. That's the way the original author had it. They had you restoring the lib and running `build`, but those steps aren't necessary. All they have to do is `dotnet restore` and `dotnet test` in the tests project. It restores both, builds both, and runs the tests.